tests/testthat/test-hard-arrange.R

```diff
@@ -0,0 +1,41 @@
+context("test-arrange")
+
+setup({
+  
+  df = disk.frame:::gen_datatable_synthetic(1e3+11)
+  data.table::fwrite(df, "tmp_pls_delete_gb.csv")
+})
+
+
+test_that("test hard_arrange on disk.frame", {
+  dff = csv_to_disk.frame("tmp_pls_delete_gb.csv", "tmp_pls_delete_gb.df")
+  
+  # Sort ascending
+  sorted_dff <- dff %>% hard_arrange(id1, id4)
```

Comment:
I am getting warning

```r
Warning message:
Using `as.character()` on a quosure is deprecated as of rlang 0.3.0.
Please use `as_label()` or `as_name()` instead.
```

We need to get rid of all warnings before submissions to CRAN.

Reply:
Rewrote that section to use rlang::quo_is_symbol. This removes the deprecation warning.